### PR TITLE
fix issue #41 #50 order["stopPrice"] is always empty

### PR DESCRIPTION
**Pull request by mingwugmail**
fix issue #41 #50 order["stopPrice"] is always empty

### CODE REVIEW

**Comment by mingwugmail** [comment]
@jessecooper  can you review this PR?

File: pyetrade/order.py [hunk 1]
```diff
@@ -156,7 +156,10 @@ def build_order_payload(self, order_type, **kwargs):
         }
         order = kwargs
         order["Instrument"] = instrument
-        order["stopPrice"] = ""
+        if "stopPrice" in kwargs:
+          stopPrice = (float(kwargs["stopPrice"]) - 0.01)
```
Comment (mingwugmail) [inline pyetrade/order.py:160]: This is to properly convert float number to 2 decimal place dollar value 2 line below.

**Comment by jessecooper** [comment]
@mingwugmail would you be comfortable adding unit tests around this functionality to make sure there are no regressions in the future?

**Comment by mingwugmail** [comment]
> @mingwugmail would you be comfortable adding unit tests around this functionality to make sure there are no regressions in the future?

@jessecooper  Done.

**Comment by mingwugmail** [comment]
> @mingwugmail would you be comfortable adding unit tests around this functionality to make sure there are no regressions in the future?

All "SELL", "SELL_SHORT", "BUY", "BUY_TO_COVER" tested.

**Comment by jessecooper** [comment]
@mingwugmail thank you for the contribution

**Pull request merged**